airflow/api_fastapi/core_api/routes/public/tasks.py

```diff
@@ -0,0 +1,54 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+from fastapi import HTTPException, Request, status
+
+from airflow.api_fastapi.common.router import AirflowRouter
+from airflow.api_fastapi.core_api.openapi.exceptions import create_openapi_http_exception_doc
+from airflow.api_fastapi.core_api.serializers.tasks import TaskResponse
+from airflow.exceptions import TaskNotFound
+from airflow.models import DAG
+
+tasks_router = AirflowRouter(tags=["Task"], prefix="/dags")
```

Comment:
Hi, let me know if using `/dags` prefix makes sense here, or if `tasks_router` should be added as a child to dags_router (or if there's another better way to do this).

Reply:
I think it's fine to do it that way, this is how we do it for other routers.

I think we can even add to the prefix the `/{dag_id}`, all tasks endpoint need it.

Reply:
Done, using prefix `/dags/{dag_id}/tasks` since it's common for both Get Task and Get Tasks endpoints.